webanno/webanno
# Feature/1197 codebook annotation support

**What's in the PR**
1. Codebook tree structure for project page
2. smaller codebook tree nodes for annotation mode

**How to test manually**
1. Run App and see in project settings > codebooks
2. Run App and see in annotation mode

**Automatic testing**

**Documentation**
